OpenChart/src/Application.cs

```diff
@@ -174,5 +193,13 @@ protected override void OnStartup()
                 Environment.Exit(1);
             }
         }
+
+        private void addAction(IAction action)
+        {
+            // FIXME: Can't add accelerators/hotkeys since the Gtk wrapper takes the wrong
+            // type of argument, resulting in a segfault.
```

Comment:
Amazing...

Reply:
I opened an issue for it on the repo so hopefully it gets resolved